application/F3DOptionsTools.cxx

```diff
@@ -192,14 +192,19 @@ void PrintHelpPair(
   std::string_view key, std::string_view help, int keyWidth = 10, int helpWidth = 70)
 {
   std::stringstream ss;
-  ss << "  " << std::left << std::setw(keyWidth) << key << " " << std::setw(helpWidth) << help;
+  ss << "  " << std::left << std::setw(keyWidth) << key;
+  if (key.size() > static_cast<size_t>(keyWidth))
+  {
+    ss << "\n  " << std::setw(keyWidth) << " "; // Indent the second line
```

Comment:
i dont think you need the keyWidth here thoug

Reply:
Sorry you are right. I added it with an instinct but it's not necessary to check the second line.